### PR TITLE
fix: Base resizing on a specifically styled div rather than table element

### PR DESCRIPTION
**Pull request by bdow**

### CODE REVIEW

File: src/components/Table/columnSizes.ts [hunk 1]
```diff
@@ -25,8 +25,8 @@ import { useDebouncedCallback } from "use-debounce";
 export function useSetupColumnSizes(
   style: GridStyle,
   columns: GridColumn<any>[],
-  tableRef: MutableRefObject<HTMLElement | null>,
-  resizeTarget: MutableRefObject<HTMLElement | null> | undefined,
+  resizeRef: MutableRefObject<HTMLDivElement | null>,
+  externalResizeTarget: MutableRefObject<HTMLElement | null> | undefined,
```
Comment (stephenh) [inline src/components/Table/columnSizes.ts:29]: Hm, should we have just 1 `resizeRef` param here, and have line 325 be:

```
const columnSizes = useSetupColumnSizes(style, columns, resizeRef ?? resizeTarget);
```

?

AFAICT we only ever use 1 or the other in this method?

Comment (bdow) [reply src/components/Table/columnSizes.ts:29]: True, will update

File: src/components/Table/GridTable.tsx [hunk 2]
```diff
@@ -297,6 +297,8 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
   // We only use this in as=virtual mode, but keep this here for rowLookup to use
   const virtuosoRef = useRef<VirtuosoHandle | null>(null);
   const tableRef = useRef<HTMLElement>(null);
```
Comment (stephenh) [inline src/components/Table/GridTable.tsx:299]: Looks like maybe we don't need `tableRef` anymore?

File: src/components/Table/GridTable.tsx [hunk 3]
```diff
@@ -513,6 +515,7 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
   return (
     <RowStateContext.Provider value={rowStateContext}>
       <PresentationProvider fieldProps={fieldProps} wrap={style?.presentationSettings?.wrap}>
+        <div ref={resizeRef} css={Css.w100.$} />
```
Comment (stephenh) [inline src/components/Table/GridTable.tsx:518]: This makes a lot of sense!

**Review by stephenh — APPROVED** [review]
Nice! Will be really interested to see if this fixes the chromatic issues.

File: src/components/Table/GridTable.tsx (hunk 2 above)
Comment (bdow) [reply src/components/Table/GridTable.tsx:299]: Yea, I don't think so... wasn't sure if I should delete it yet though. I guess I can.

Comment (bdow) [reply src/components/Table/GridTable.tsx]: Alrighty - since you took the time to see if it was okay to delete, I deleted it.

Comment (stephenh) [reply src/components/Table/GridTable.tsx]: :100: :-)

**Pull request merged**